frontend/src/lib/i18n/en.json

```diff
@@ -35,6 +35,37 @@
     "otherLanguages": "Other Languages",
     "themes": "Themes"
   },
+  "icons": {
```

Comment:
weren't these already defined in Strapi?

Reply:
I don't think so, but we can remove them all now anyway.